docs/content/latest/quick-start/build-apps/go/ysql-pg.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Build a Go application that uses YSQL
+headerTitle: Build a Go application
+linkTitle: Go
+description: Build a sample Go application with the Go PostgreSQL driver and perform basic database operations with YugabyteDB.
+menu:
+  latest:
+    parent: build-apps
+    name: Go
+    identifier: go-3
+    weight: 552
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../ysql-pgx/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PGX
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pg/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PG
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [Go-pg client and ORM](https://pkg.go.dev/github.com/go-pg/pg), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register) to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.8](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+You can choose to enable or disable SSL for your local YugabyteDB cluster. Refer [here](../../../../secure/tls-encryption/client-to-server/) to learn about configuring SSL/TLS for your YugabyteDB cluster. Yugabyte Cloud requires SSL connections, and SSL/TLS is enabled by default for client-side authentication.
+
+#### CA certificate
+
+Create a [CA certficate](../../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) to launch your YugabyteDB cluster with SSL/TLS enabled.
```

Comment:
It may sound like this is an additional step. Instead we can say something like 'Use the CA certificate generated above as part of the SSL/TLS configuration of your cluster. Same in other two driver docs.